src/sass/variables.scss

```diff
@@ -1,9 +1,19 @@
 
 // Global Colors
 $dh-color-white: white;
+$dh-color-gray80: #cccccc;
 
+$dh-link_color-default: #005ea2;
```

Comment:
did you need to add the link colors? Were they not in the USWDS css already?

Reply:
The default color that I got was #1400EE.